src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,54 @@
+package view;
+
+import domain.Ladder;
+import domain.Line;
+import domain.Names;
+import domain.Point;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OutputView {
+
+    private static final String EDGE_OF_POINT = "|";
+    private static final String PASSABLE_POINT = "-----";
+    private static final String BLOCKED_POINT = "     ";
+    private static final String ERROR_MESSAGE_PREFIX = "[ERROR] ";
+
+    public void printResult(Names names,
+                            Ladder ladder) {
+        System.out.println("실행결과");
+
+        System.out.println(getFormattedNames(names));
+        printLadder(ladder);
+    }
+
+    private String getFormattedNames(Names names) {
+        return names.getNames().stream()
+                .map(name -> String.format("%-5s", name.getName()))
+                .collect(Collectors.joining(" "));
```

Comment:
Names 일급컬렉션에 stream() 이라는 메서드를 구현해보면 어떨까요? 
`일급컬렉션이 자바의 컬렉션과 최대한 비슷하게 사용할 수 있다면 좋지않을까~` 라는 취지인데 포이는 어떻게 생각하시나요?
가령 `코드 읽는 사람이 일급 컬렉션인줄 몰라서 위험하지않을까?` 같은 의견도 있어요 😄 

```suggestion
        return names.stream()
                .map(name -> String.format("%-5s", name.getName()))
                .collect(Collectors.joining(" "));
```

Reply:
일급 컬렉션도 결국 컬렉션의 역할을 하기 위한 것이란 것을 잊고 있었습니다. 찰리의 의견대로 바꿨더니 코드가 훨신 직관적으로 변한 것 같아요!

**두 개에 단어가 합쳐져 만들어진 단어에서는 뒷 단어가 이의 본질이다** 라는 말이 생각나네요ㅎㅎ